### PR TITLE
Update OAuth2

### PR DESCRIPTION
**Pull request by blootsvoets**
Adds `redirect_uri` whenever requesting an OAuth 2.0 access token. This is now required by e.g. the Fitbit API.

### CODE REVIEW

**Review by nivemaham — COMMENTED** [review]
i can't see the OAuth2 part

**Comment by blootsvoets** [comment]
It's mainly in OAuth2RestSourceAuthorizationService.kt, but also in Config.kt

File: authorizer-app-backend/build.gradle.kts [hunk 1]
```diff
@@ -11,6 +12,10 @@ plugins {
 
 application {
     mainClass.set("org.radarbase.authorizer.Main")
+    applicationDefaultJvmArgs = listOf(
+        "-Djava.security.egd=file:/dev/./urandom",
```
Comment (nivemaham) [inline authorizer-app-backend/build.gradle.kts:16]: Why do we use this? Just for my interest.

**Review by nivemaham — APPROVED** [review]
LGTM

File: authorizer-app-backend/build.gradle.kts (hunk 1 above)
Comment (blootsvoets) [reply authorizer-app-backend/build.gradle.kts:16]: It tells Java that it shouldn't rely on the entropy of the machine (on VM's there is typically too little entropy), but use a pseudorandom generator. This avoids any delays in SSL processing where randomness is required -- see e.g. https://www.baeldung.com/java-security-egd#10-what-about-virtual-machines

**Pull request merged**